### PR TITLE
Z-score Threshold tutorial 

### PR DESCRIPTION
**Pull request by samuel-wj-chapman**
## Pull Request Description:
notebook tutorial guiding the user through the use of z-score and its affect on quantisation thresholds and accuracy 

## Checklist before requesting a review:
- [x] I set the appropriate labels on the pull request.
- [x] I have added/updated the release note draft (if necessary).
- [x] I have updated the documentation to reflect my changes (if necessary).
- [x] All function and files are well documented. 
- [x] All function and classes have type hints.
- [x] There is a licenses in all file.
- [x] The function and variable names are informative. 
- [x] I have checked for code duplications.
- [x] I have added new unittest (if necessary).

### CODE REVIEW

File: tutorials/notebooks/keras/ptq/example_keras_activation_z_score_threshold.ipynb [hunk 1]
```diff
@@ -0,0 +1,847 @@
...
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "a22e6d68-c40f-40bf-ab74-ff453011aeac",
+      "metadata": {
+        "id": "a22e6d68-c40f-40bf-ab74-ff453011aeac"
+      },
+      "outputs": [],
+      "source": [
+        "#print layer name\n",
+        "for index, layer in enumerate(float_model.layers):\n",
```
Comment (elad-c) [inline tutorials/notebooks/keras/ptq/example_keras_activation_z_score_threshold.ipynb:427]: wouldn't `print(float_model.layers[51].name)` do the trick?

File: tutorials/notebooks/keras/ptq/example_keras_activation_z_score_threshold.ipynb [hunk 2]
```diff
@@ -0,0 +1,847 @@
...
+        "MCT has the option to remove these using z-score thresholding. Allowing the user to remove data based on standard distributions.\n",
+        "\n",
+        "The Z-score of a value is calculated by subtracting the mean of the dataset from the value and then dividing by the standard deviation of the dataset. This measures how many standard deviations an element is from the mean.\n",
+        "\n",
+        "\n",
+        "\n",
+        "To calculate a threshold $t$ for quantization based on a Z-score threshold $Z_t$, you might define $t$ as a function of $Z_t$, $\\mu$, and $\\sigma$, such as:\n",
+        "\n",
+        "$$\n",
+        "t(Zt)=\\frac{μ+Zt⋅σ}{μ+Zt​⋅σ}\n",
```
Comment (elad-c) [inline tutorials/notebooks/keras/ptq/example_keras_activation_z_score_threshold.ipynb:77]: isn't this just `\\frac{μ+Zt⋅σ}{μ+Zt​⋅σ}` = 1?

**Review by elad-c — CHANGES_REQUESTED** [review]
in general:

1. in comments: make sure you have a space after the "#" character
2. although "quantise" is obviously correct, maybe change to "quantize" to be consistent with out code?
3. it's clearer to write z-score (instead of "zscore"

**Comment by samuel-wj-chapman** [comment]
made all the relevant changes @elad-c  thanks for feed back

File: tutorials/notebooks/keras/ptq/example_keras_activation_z_score_threshold.ipynb (hunk 1 above)
Comment (elad-c) [reply tutorials/notebooks/keras/ptq/example_keras_activation_z_score_threshold.ipynb:429]: ?

**Review by elad-c — APPROVED** [review]
with 1 little comment

File: tutorials/notebooks/keras/ptq/example_keras_activation_z_score_threshold.ipynb (hunk 1 above)
Comment (samuel-wj-chapman) [reply tutorials/notebooks/keras/ptq/example_keras_activation_z_score_threshold.ipynb]: sorry, miss read your original comment. fixed

**Review by elad-c — APPROVED** [review]

**Pull request merged**